--- dbus-sharp-0.8.1/src/Makefile.am.orig	2022-04-25 21:00:29.442324896 +0200
+++ dbus-sharp-0.8.1/src/Makefile.am	2022-04-25 21:05:16.230771229 +0200
@@ -53,14 +53,14 @@ $(TARGET): $(CSFILES) AssemblyInfo.cs
 
 install-data-local: $(TARGET)
 	@if test -n '$(TARGET)'; then                       \
-          echo "$(GACUTIL) /i $(TARGET) /package $(PACKAGE)-$(API_VERSION) /f /gacdir $(libdir) /root $(DESTDIR)$(libdir)";    \
-          $(GACUTIL) /i $(TARGET) /package $(PACKAGE)-$(API_VERSION) /f /gacdir $(libdir) /root $(DESTDIR)$(libdir) || exit 1; \
+          echo "$(GACUTIL) /i $(TARGET) /package $(PACKAGE)-$(API_VERSION) /f /gacdir $(prefix)/lib /root $(DESTDIR)$(prefix)/lib";    \
+          $(GACUTIL) /i $(TARGET) /package $(PACKAGE)-$(API_VERSION) /f /gacdir $(prefix)/lib /root $(DESTDIR)$(prefix)/lib || exit 1; \
         fi
 
 uninstall-local:
 	@if test -n '$(TARGET)'; then                       \
-          echo "$(GACUTIL) /u $(ASSEMBLY) /package $(PACKAGE)-$(API_VERSION) /gacdir $(libdir) /root $(DESTDIR)$(libdir)";  \
-          $(GACUTIL) /u $(ASSEMBLY) /package $(PACKAGE)-$(API_VERSION) /gacdir $(libdir) /root $(DESTDIR)$(libdir) || exit 1;   \
+          echo "$(GACUTIL) /u $(ASSEMBLY) /package $(PACKAGE)-$(API_VERSION) /gacdir $(prefix)/lib /root $(DESTDIR)$(prefix)/lib";  \
+          $(GACUTIL) /u $(ASSEMBLY) /package $(PACKAGE)-$(API_VERSION) /gacdir $(prefix)/lib /root $(DESTDIR)$(prefix)/lib || exit 1;   \
         fi
 
 .PHONY: $(TARGET)
